packages/@aws-cdk/aws-ec2/test/instance.test.ts

```diff
@@ -2,10 +2,10 @@ import { expect, haveResource } from '@aws-cdk/assert';
 import { StringParameter } from '@aws-cdk/aws-ssm';
 import * as cxschema from '@aws-cdk/cloud-assembly-schema';
 import { Stack } from '@aws-cdk/core';
-import { Test } from 'nodeunit';
+import { nodeunitShim, Test } from 'nodeunit-shim';
 import { AmazonLinuxImage, BlockDeviceVolume, EbsDeviceVolumeType, Instance, InstanceClass, InstanceSize, InstanceType, Vpc } from '../lib';
 
-export = {
+nodeunitShim({
```

Comment:
Btw why do we need to wrap each test with this? A simple `require` of the test file and producing a jest test for each exported function should have done the trick. No?

So basically you could just added a single jest test file inside the `test` directory which would find all nodeunit tests and export them as jest tests.

Reply:
Potentially.

Though we'd also need to hook the `require("nodeunit")` call.

I'd encourage you to try to do it that way when you go to convert a module :)